packages/client/lib/execution/vmexecution.ts

```diff
@@ -15,8 +15,7 @@ import { Event } from '../types'
 import { Execution, ExecutionOptions } from './execution'
 import { ReceiptsManager } from './receipt'
 import type { Block } from '@ethereumjs/block'
-import type { RunBlockOpts } from '@ethereumjs/vm/dist/runBlock'
-import type { TxReceipt } from '@ethereumjs/vm/dist/types'
+import type { RunBlockOpts, TxReceipt } from '@ethereumjs/vm'
```

Comment:
Oh, that wasn't apparent to me that these things get so much nicer out of the box and many places going down to one-line imports. Really cool! 😄